### PR TITLE
Long node text fix

### PR DESCRIPTION
**Pull request by satya210**
Issue:
Screenshot 2023-06-30 at 5 10 59 PM

After fix:
Screenshot 2023-06-29 at 12 29 52 PM

### CODE REVIEW

File: src/collapsible_tree/collapsible_tree.ts [hunk 1]
```diff
@@ -213,7 +213,19 @@ const vis: CollapsibleTreeVisualization = {
...
+          (d: any) => { 
+            const distance = d.y - (d.parent?.y || 0)
+            const charSize = (parseFloat(getComputedStyle(element).fontSize) / 3) || 3
+
+            let maxCharWidth = Math.floor(distance / charSize);
+            if(maxCharWidth === 0) {
+                return d.data.name
+            }
+            
+            maxCharWidth -= 4
```
[the remark this answers is not in the document]
Comment (satya210) [reply src/collapsible_tree/collapsible_tree.ts]: @yesudeep I've made the suggested changes. Please review it.

[the remark this answers is not in the document]
Comment (satya210) [reply src/collapsible_tree/collapsible_tree.ts]: Thanks 👍

**Comment by satya210** [comment]
adding the test results SS. cc: @yesudeep 
Screenshot 2023-07-31 at 1 09 09 PM

**Comment by ShenoyPraveen** [comment]
Hey @andres-rubio-go @yesudeep is there update on this PR? would it possible to merge if things look good?

**Comment by ShenoyPraveen** [comment]
@andres-rubio-go could you please share your findings?

**Comment by satya210** [comment]
@andres-rubio-go @yesudeep Any updates on the PR?

**Comment by satya210** [comment]
andres-rubio-go can you check again with the latest commit. There was some bug. I fixed it.

**Comment by satya210** [comment]
Hi @andres-rubio-go I've made some changes to the numbers in the last commit w.r.t. this issue. Could you please check with the latest commit?

**Comment by andres-rubio-go** [comment]
Tested last commit on testing dashboard and fix looks good now, I'll confirm if we need to do some regression testing just to make sure nothing else broke but other than that this looks good to me. I attach screenshots:

Screenshot 2023-09-01 11 47 47 AM
4dad49a-16c8-4040-9970-901fd7fbea59)
Screenshot 2023-09-01 11 46 22 AM

**Pull request merged**